src/App.tsx

```diff
@@ -17,35 +37,29 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
+          <NavItem path="/" title="Home" />
+          <NavItem path="/tabs" title="Tabs" />
         </div>
       </div>
     </nav>
 
     <div className="section">
       <div className="container">
-        <h1 className="title">Home page</h1>
-        <h1 className="title">Tabs page</h1>
-        <h1 className="title">Page not found</h1>
+        <Routes>
+          <Route
+            path="/"
+            element={<h1 className="title">Home page</h1>}
```

Comment:
make separate component for it
```suggestion
            element={...}
```